src/main/charts/common/templates/_jmx.tpl

```diff
@@ -0,0 +1,57 @@
+{{/*
+Jmx config volume
+*/}}
+{{- define "common.jmx.config.volume" -}}
+{{- if .Values.monitoring.exposeJmxMetrics }}
+- name: jmx-config
+  configMap:
+    name: {{ include "common.names.fullname" . }}-jmx-config
+{{- end }}
+{{- end }}
+
+{{/*
+Jmx config volume mount
+*/}}
+
+{{- define "common.jmx.config.volumeMounts" -}}
+{{- if .Values.monitoring.exposeJmxMetrics }}
+- name: jmx-config
+  mountPath: /opt/atlassian/confluence/jmx
```

Comment:
can we use confluence in the path if this is for all DC products?

Reply:
Good catch! Even though it's not a big deal and the mount will happen anyways, it needs to be smth generic. I'll fix that!

Reply:
Fixed. It's just `/opt/atlassian/jmx` now